src/__tests__/looksrare/executeMultipleTakerBids.test.ts

```diff
@@ -118,7 +118,7 @@ describe("execute multiple taker bids", () => {
       { maker: makers[0], taker: taker1, signature: signature, merkleTree: merkleTreeProofs[0] },
       { maker: makers[1], taker: taker2, signature: signature, merkleTree: merkleTreeProofs[1] },
     ];
-    const user1InitialBalance = await signers.user1.getBalance();
+    const user1InitialBalance = await ethers.provider.getBalance(signers.user1.address);
```

Comment:
`getBalance` is not existing in user1 signer, which was injected by `hardhat-ethers`, so I used the different approach